code/src/components/Survey.js

```diff
@@ -0,0 +1,368 @@
+                    text={rainyFood ? 'Next Baby!' : 'Choose something!'}
+                    click={nextQuestion}
+                    disable={!rainyFood}
+                  />
+                </div>
+                <Progressbar
+                  value='30'/>
+              </>
+            )}
+            {question === 2 && (
+              <>
+                <Dropdown
+                  selectValue={restaurantHunt}
+                  setSelectValue={setRestaurantHunt}
+                  question={'What is your process of choosing a restaurant?'}
+                  optionsArray={restaurantMethodArray}
+                /><div className="button-container">
+                  <Button
+                    type='button'
+                    text='Previous Pretty Please!'
+                    click={previousQuestion}
+                  />
+                  <Button
+                    type='button'
+                    text={restaurantHunt ? 'Next Baby!' : 'Choose something!'}
+                    click={nextQuestion}
+                    disable={!restaurantHunt}
+                  />
+                </div>
+                <Progressbar
+                  value='45'/>
+              </>
+            )}
+            {question === 3 && (
+              <>
+                <Dropdown
+                  selectValue={saladMaker}
+                  setSelectValue={setSaladMaker}
+                  question={'You are making a salad. What is the main ingrediens?'}
+                  optionsArray={saladArray}
+                /><div className="button-container">
+                  <Button
+                    type='button'
+                    text='Previous Pretty Please!'
+                    click={previousQuestion}
+                  />
+                  <Button
+                    type='button'
+                    text={saladMaker ? 'Next Baby!' : 'Choose something!'}
+                    click={nextQuestion}
+                    disable={!saladMaker}
+                  />
+                </div>
+                <Progressbar
+                  value='60'/>
+              </>
+            )}
+            {question === 4 && (
+              <>
+                <Radio
+                  ageGroup={ageGroup}
+                  setAgeGroup={setAgeGroup}
+                  question={'Thanks for doing this survey! Please check your agegroup:'}
+                  radioButtonsArray={[
+                    {
+                      value: '0-20',
+                      message: 'Teeny tiny baby'
+                    },
+                    {
+                      value: '21-40',
+                      message: 'Young and fresh baby'
+                    },
+                    {
+                      value: '41-60',
+                      message: 'Prime years baby'
+                    },
+                    {
+                      value: '61-100',
+                      message: 'Oldie but goldie baby'
+                    }
+                  ]}
```

Comment:
Couldn't this also be an array with object that you declare in the same way as you did with the other array's?